### PR TITLE
Add installation messages

### PR DESCRIPTION
**Pull request by Bishoy-at-pieces**

### CODE REVIEW

**Review by caleb-at-pieces — COMMENTED** [review]
looking good to me, probably want to loop @mack-at-pieces in on this

**Comment by Bishoy-at-pieces** [comment]
Seems like there is 2 readme files something wired

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+
+Enter the editor mode with `command/ctrl + e`. You can then modify your asset and save it using `command/ctrl + s`.
+
+#### Copy an Asset
+
+If you're in an asset sheet, you can quickly copy the asset content without selecting it using `command/ctrl + c`.
+
+### Copilot Ask
+
+Need the copilot to comment, fix a bug, or add a doc string? Simply go to the command palette and run `Pieces: Ask Copilot`.
```
Comment (mark-at-pieces) [inline README.md:27]: do we have this feature in the sublime plugin?

File: README.md [hunk 2]
```diff
@@ -1 +1,33 @@
-# plugin_sublime
\ No newline at end of file
+![Pieces Sublime Plugin](https://camo.githubusercontent.com/69c990240f877927146712d45be2f690085b9e45b4420736aa373917f8e0b2c8/68747470733a2f2f73746f726167652e676f6f676c65617069732e636f6d2f7069656365735f7374617469635f7265736f75726365732f7066645f77696b692f5049454345535f4d41494e5f4c4f474f5f57494b492e706e67)
+
+# Pieces Sublime Plugin
```
Comment (mark-at-pieces) [inline README.md:3]: do we have the ability to right click, save an asset as well? and are there any other features in here that we are listing like search? that we might want to add as well?

**Review by mark-at-pieces — COMMENTED** [review]
just added some thoughts about the readme, looking good so far might want to add all the features tho correct?

File: README.md (hunk 1 above)
Comment (Bishoy-at-pieces) [reply README.md:27]: https://github.com/pieces-app/plugin_sublime/pull/21

Just the refactor command

File: README.md (hunk 2 above)
Comment (Bishoy-at-pieces) [reply README.md:3]: We have the ability to right click to create an asset

**Review by mark-at-pieces — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -1 +1,54 @@
...
+
+
+##### <p align="center"> [Website](https://pieces.app/) • [Pieces OS Documentation](https://docs.pieces.app/) • [Pieces Sublime github repository](https://github.com/pieces-app/plugin_sublime)
+</p>
+
+
+## Getting Started
+
+Welcome to the Pieces Sublime Plugin.
+Pieces Sublime plugin robust plugin designed to manage your pieces snippets and interact seamlessly with the Pieces OS client.
```
Comment (mack-at-pieces) [inline README.md:11]: "The Pieces Sublime plugin is designed to manage your saved materials in Pieces right from within Sublime, and interact seamlessly with Pieces OS."

File: README.md [hunk 4]
```diff
@@ -1 +1,54 @@
...
+To get started, navigate to `Preferences > Package Settings > Pieces > Keybindings` to check out and modify the keybindings. If you wish to change the llm model you're using or the host, you can do so in the settings.
```
Comment (mack-at-pieces) [inline README.md:12]: Is the first thing we want them to do change the keybindings?

Also - is the copilot available? If not, we probably don't need a setting to change the LLM quite yet. Or am I misunderstanding?

File: README.md [hunk 5]
```diff
@@ -1 +1,54 @@
...
+
+### Open Asset Command
```
Comment (mack-at-pieces) [inline README.md:14]: Lets use the same terminology we use across other documentation, which is typically "Saved Materials".

"Open & View Saved Materials"

File: README.md [hunk 6]
```diff
@@ -1 +1,54 @@
...
+
+To open your assets, simply open your command palette and run `Pieces: Open an asset`. Then, select the asset you wish to open.
```
Comment (mack-at-pieces) [inline README.md:16]: Can replace "assets" in here with "saved materials".

Also - are we able to adjust the string on the command? "Pieces: Open Saved Material"?

File: README.md [hunk 7]
```diff
@@ -1 +1,54 @@
...
+
+#### Editor Mode
```
Comment (mack-at-pieces) [inline README.md:18]: Edit Saved Materials

File: README.md [hunk 8]
```diff
@@ -1 +1,54 @@
...
+
+Enter the editor mode with `command/ctrl + e`. You can then modify your asset and save it using `command/ctrl + s`.
```
Comment (mack-at-pieces) [inline README.md:20]: To edit a saved material, you'll need to toggle into editor mode. You can do so by tapping `command/ctrl+e`. From there, you will be able to modify the contents of your material and save the changes using `command/ctrl + s`.

File: README.md [hunk 9]
```diff
@@ -1 +1,54 @@
...
+
+#### Copy an Asset
```
Comment (mack-at-pieces) [inline README.md:22]: Copy Saved Material

File: README.md [hunk 10]
```diff
@@ -1 +1,54 @@
...
+
+If you're in an asset sheet, you can quickly copy the asset content without selecting it using `command/ctrl + c`.
```
Comment (mack-at-pieces) [inline README.md:24]: What does it mean to be in an "asset sheet" ?

File: README.md (hunk 1 above)
Comment (mack-at-pieces) [reply README.md:28]: If we don't have this feature, we should strike this bullet or make it more explicit about what the feature is. "Refactor Code" or something.

Comment (mack-at-pieces) [reply README.md:28]: & how does this feature work exactly? Is there a demo video somewhere I can check out?

File: README.md [hunk 11]
```diff
@@ -1 +1,54 @@
...
+
+### Copilot Ask
+
+Need the copilot to comment, fix a bug, or add a doc string? Simply go to the command palette and run `Pieces: Ask Copilot`.
+
+### Login/Logout
+
+You can log in and out of your Pieces account using the `Pieces: Login` or `Pieces: Logout` command in your command palette. An output panel is available to check your cloud status and login status.
+
+### Right click to add an asset
```
Comment (mack-at-pieces) [inline README.md:34]: "Save a Snippet" -> would move this one closer to the top of the document.

File: README.md (hunk 1 above)
Comment (mack-at-pieces) [reply README.md:28]: Would move this one closer to the top of the document.

File: README.md [hunk 12]
```diff
@@ -1 +1,54 @@
...
+
+You can create a new asset to pieces select the text, right click then choose `pieces>save to pieces`.
```
Comment (mack-at-pieces) [inline README.md:36]: "To save a new snippet in Pieces, select the code you wish to save, right-click to open the context menu, and select "Pieces > Save to Pieces".

File: README.md [hunk 13]
```diff
@@ -1 +1,54 @@
...
+
+### Connect
```
Comment (mack-at-pieces) [inline README.md:38]: Connect with the Pieces Community

File: README.md [hunk 14]
```diff
@@ -1 +1,54 @@
...
+
+#### Join our [Discord Community](https://discord.gg/getpieces)
+
+Become a part of our Discord community to stay updated and engage in discussions about our features.
+
+#### Feature Request
```
Comment (mack-at-pieces) [inline README.md:44]: Submit a Feature Request or Feedback

File: messages/install.txt [hunk 15]
```diff
@@ -0,0 +1,12 @@
+Welcome to Pieces sublime Plugin
```
Comment (mack-at-pieces) [inline messages/install.txt:1]: Welcome to the Pieces Sublime Plugin

File: messages/install.txt [hunk 16]
```diff
@@ -0,0 +1,12 @@
...
+
+
+Please make sure you have the pieces os https://docs.pieces.app/installation-getting-started/what-am-i-installing
```
Comment (mack-at-pieces) [inline messages/install.txt:4]: To get started, please make sure you have Pieces OS installed, running, and up-to-date. 

Pieces OS is the centralized storage agent that works on-device, unifying various developer tools (including the Pieces for Developers Desktop Dpp) to proactively capture and enrich useful materials, streamline collaboration, and solve complex problems through a contextual understanding of your unique workflow.

[Get Pieces OS here](https://docs.pieces.app/installation-getting-started/what-am-i-installing)

File: messages/install.txt [hunk 17]
```diff
@@ -0,0 +1,12 @@
...
+
+
+
+Useful links:
```
Comment (mack-at-pieces) [inline messages/install.txt:8]: Lets also add our YouTube here

**Review by mack-at-pieces — CHANGES_REQUESTED** [review]
Left some comments. Please take a look. It will also be helpful to read through the Pieces for VS Code documentation to get a sense of how we like to talk about and describe the products/plugins and their features.

This will work for now, but we'll ultimately want to work with Cole, Mason, and the growth team to bring this more in line with some of our other README's 👍🏼

File: README.md (hunk 4 above)
Comment (Bishoy-at-pieces) [reply README.md:12]: So the keybindings can be changed if they want to but not I don't think they should

Also we have the refactor command which will use the ask endpoint. I can share the video but I think you saw it before.

File: README.md (hunk 10 above)
Comment (Bishoy-at-pieces) [reply README.md:24]: image
In sublime context this is called html sheet so what should I say here?

File: README.md [hunk 18]
```diff
@@ -1 +1,55 @@
...
+
+
+##### <p align="center"> [Website](https://pieces.app/) • [Pieces OS Documentation](https://docs.pieces.app/) • [Pieces Sublime github repository](https://github.com/pieces-app/plugin_sublime)
+</p>
+
+
+## Getting Started
+
+Welcome to the Pieces Sublime Plugin.
+The Pieces Sublime plugin is designed to manage your saved materials in Pieces right from within Sublime, and interact seamlessly with Pieces OS.client.
```
Comment (mack-at-pieces) [inline README.md:11]: Don't need the `.client` here at the end 👍🏼

File: README.md [hunk 19]
```diff
@@ -1 +1,55 @@
...
+
+To get started, Check the settings. If you wish to change the llm model you're using or the host, you can do so in the settings `Preferences > Package Settings > Pieces > Settings`.
```
Comment (mack-at-pieces) [inline README.md:13]: Lowercase "c" in "check".

File: README.md (hunk 10 above)
Comment (mack-at-pieces) [reply README.md:25]: "If you're viewing a saved material.."

File: README.md [hunk 20]
```diff
@@ -1 +1,55 @@
...
+
+Need the copilot to comment, fix a bug, or add a doc string? Simply go to the command palette and run `Pieces: Ask Copilot`.
+
+### Login/Logout
+
+You can log in and out of your Pieces account using the `Pieces: Login` or `Pieces: Logout` command in your command palette. An output panel is available to check your cloud status and login status.
+
+### Connect with the Pieces Community
+
+#### Join our [Discord Community](https://discord.gg/getpieces)
```
Comment (mack-at-pieces) [inline README.md:41]: Can we link to the Pieces YouTube here and have it be like "View Power Tips & Best Practices"

Comment (mack-at-pieces) [reply README.md:41]: (can we add a section that links to our YouTube)**

**Review by mack-at-pieces — CHANGES_REQUESTED** [review]

File: README.md [hunk 21]
```diff
@@ -1 +1,59 @@
...
+
+Stay connected and up-to-date with our latest [blogs](https://code.pieces.app/blog).
+
+#### Plugins
+
+Explore our collection of awesome Pieces [plugins](https://code.pieces.app/plugins).
+
+#### View Power Tips & Best Practices
+
+Don't forgot to check out our youtube channel for [Power Tips & Best Practices](https://youtube.com/@getpieces)
```
Comment (mack-at-pieces) [inline README.md:59]: "Don't forget to check out our YouTube channel..." Few typos in here.

File: messages/install.txt [hunk 22]
```diff
@@ -0,0 +1,19 @@
...
+and solve complex problems through a contextual understanding of your unique workflow.
+
+Get Pieces OS -> https://docs.pieces.app/installation-getting-started/what-am-i-installing
+
+
+
+Useful links:
+-------------
+Website: https://pieces.app
+Github: https://github.com/pieces-app/plugin_sublime
```
Comment (mack-at-pieces) [inline messages/install.txt:17]: These are technically "GitHub" and "YouTube"

**Review by mack-at-pieces — CHANGES_REQUESTED** [review]

**Review by mack-at-pieces — APPROVED** [review]

**Pull request merged**